GetWowpediaInfo.lua

```diff
@@ -208,24 +323,84 @@ end
 
 table.sort(apiKeys)
 
+out.write(out, "--- @class unknown @ unknown type\n\n")
+
 for _,k in pairs(apiKeys) do
+    local preFunction = ""
     local functionName = k
-    local returnValues
-    local argValues
+    local argValues = ""
+
+    if api_entries[k].description then
+        preFunction = preFunction .. "--- ### " .. api_entries[k].description .. "\n"
+    end
 
-    if api_entries[k].returns_all then
-        returnValues = api_entries[k].returns_all
-    else
-        returnValues = "unknown"
+    if api_entries[k].address then
+        preFunction = preFunction .. "--- [" .. BASE_WOWPEDIA_ADDRESS .. api_entries[k].address .. "]\n"
+    end
+
+    -- Display return types and description
+    --if api_entries[k].returns then
+    --    for _, ret in ipairs(api_entries[k].returns) do
+    --        if api_entries[k].returnDetails[ret] then
+    --            local returnDetails = api_entries[k].returnDetails[ret]
+    --            local returnDetailsExtra = nil
+    --            local newLineStart,_ = string.find(returnDetails,"\n")
+    --            if newLineStart then
+    --                returnDetailsExtra = string.sub(returnDetails, newLineStart + 1)
+    --                returnDetails = string.sub(returnDetails, 1, newLineStart - 1)
+    --                returnDetailsExtra = "--- ```\n--- " .. returnDetailsExtra:gsub("\n", "\n--- ") .. "\n--- ```\n"
+    --            end
+    --            preFunction = preFunction .. "--- \n" .. "--- **" .. ret .. "** - _" .. api_entries[k].returnTypes[ret] .. "_ - " .. returnDetails .. "\n"
+    --            if returnDetailsExtra then
+    --                preFunction = preFunction .. returnDetailsExtra
+    --            end
+    --        end
+    --    end
+    --end
```

Comment:
If this code is not needed I would prefer it to be removed rather than having commented code added.